compiler/pota-quantization-value-test/compare_tensors.py

```diff
@@ -68,7 +68,7 @@ def compare_quantization(tensor, tensor_name, expect_dir):
     for key in json_load:
         if key == "weights":
             expected_weights = np.array(json_load["weights"])
-            input_weights = tensor["weights"][:]
```

Comment:
Just curious, where can I find this grammar reference?

Reply:
You can find it here https://docs.h5py.org/en/stable/high/dataset.html#reading-writing-data.

Reply:
@llFreetimell https://docs.h5py.org/en/stable/high/dataset.html#reading-writing-data